src/EPR.PRN.Backend.API.Common/Enums/EvidenceStatusCode.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+namespace EPR.PRN.Backend.API.Common.Enums
+{
+    public enum EvidenceStatusCode
+    {
```

Comment:
The backend has a similar enumeration that does not tie up, are you sure your values are correct?
   public enum EprnStatus
   {
       ACCEPTED  = 1,
       REJECTED,
       CANCELLED,
	AWAITINGACCEPTANCE
   }